resolwe/flow/managers/workload_connectors/slurm.py

```diff
@@ -0,0 +1,65 @@
+""".. Ignore pydocstyle D400.
+
+===============
+Slurm Connector
+===============
+
+"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import logging
+import os
+import shlex
+import subprocess
+
+from resolwe.utils import BraceMessage as __
+
+from .base import BaseConnector
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class Connector(BaseConnector):
+    """Slurm-based connector for job execution."""
+
+    def submit(self, data, dest_dir, argv, verbosity=1):
+        """Run process with SLURM.
+
+        For details, see
+        :meth:`~resolwe.flow.managers.workload_connectors.base.BaseConnector.submit`.
+        """
+        limits = data.process.get_resource_limits()
+        logger.debug(__(
+            "Connector '{}.{}' running {}.",
+            self.__class__.__module__,
+            self.__class__.__name__,
+            repr(argv)
+        ))
+        try:
+            script_path = os.path.join(dest_dir, 'slurm.sh')
+            with open(script_path, 'wt') as script:
+                script.write('#!/bin/bash\n')
+
+                # We add 200MB to the memory limit to account for
+                # executor overhead, since the executor is running in
+                # the same environment as the process.
+                script.write('#SBATCH --mem={}M\n'.format(limits['memory'] + 200))
```

Comment:
Why 200MB? Should this be a module-level constant (e.g., `EXECUTOR_MEMORY_OVERHEAD`) instead?

Reply:
@dblenkus ?

Reply:
It's just an arbitrary chosen amount that is big enough to work without problems. As clusters normally have enough memory, it is safer to use a bit larger number.

You can put it into a constant, but it shouldn't be controllable in settings.